dataland-frontend/src/components/forms/parts/UploadReports.vue

```diff
@@ -1,71 +1,73 @@
 <template>
-  <div v-if="isEuTaxonomy" class="col-3 p-3 topicLabel">
-    <h4 class="anchor title">Upload company reports</h4>
-    <p>Please upload all relevant reports for this dataset in the PDF format.</p>
-  </div>
-  <!-- Select company reports -->
-  <div :class="isEuTaxonomy ? 'col-9 formFields' : 'formField'">
-    <h3 class="mt-0">Select company reports</h3>
-    <UploadDocumentsForm
-      ref="uploadDocumentsForm"
-      @updatedDocumentsSelectedForUpload="handleUpdatedDocumentsSelectedForUpload"
-      :name="name"
-    />
-  </div>
+  <div :data-test="`upload-reports-${name}`">
```

Comment:
Here just this test-marker was introduced, correct?

Reply:
Yes.